__tests__/partials.ts

```diff
@@ -30,19 +30,35 @@ test('it allows for injecting an HBS partial', async () => {
     },
   });
   const result = await build(temp.dir, {
-    partialDirectory: temp.path('partials'),
+    partialDirectory: [temp.path('partials')],
   });
   const html = getHtmlSource(result);
 
   expect(html).toContain('<h1>Title</h1>');
 });
 
+test('it allows for injecting an HBS partial with multiple partial folders', async () => {
+  const temp = await factory.createStructure({
+    'index.html': '{{> header }}{{> header2 }}',
+    partials: {
+      'header.hbs': '<h1>Title</h1>',
+      'header2.hbs': '<h1>Different title</h1>',
+    },
+  });
+  const result = await build(temp.dir, {
+    partialDirectory: [temp.path('partials')],
+  });
+  const html = getHtmlSource(result);
```

Comment:
I think this actually tests only one partial directory; we'll want something like this instead

```suggestion
  const temp = await factory.createStructure({
    'index.html': '{{> header }}{{> header2 }}',
    partials1: {
      'header.hbs': '<h1>Title</h1>',
    },
    partials2: {
      'header2.hbs': '<h1>Different title</h1>',
    },
  });
  const result = await build(temp.dir, {
    partialDirectory: [temp.path('partials1'), temp.path('partials2')],
  });
  const html = getHtmlSource(result);
```